src/storage/txn/scheduler.rs

```diff
@@ -283,10 +286,10 @@ impl<L: LockManager> SchedulerInner<L> {
 /// Scheduler which schedules the execution of `storage::Command`s.
 #[derive(Clone)]
 pub struct Scheduler<E: Engine, L: LockManager> {
-    // `engine` is `None` means currently the program is in scheduler worker threads.
-    engine: Option<E>,
```

Comment:
It doesn't seem to be used at all.

Reply:
Right. But we need to keep the type, so `with_tls_engine` can know what the actual type is.